### PR TITLE
Fix/frontend

### PR DESCRIPTION
**Pull request by WillisLi**
What did we do: Changed foreign key, `user_id`, from `key` field to `okta` for all instances.
Files Changed: All routers, user models, and user tables.
Why was this change necessary: To prepare for connection between backend and frontend.

[Loom Video](https://www.loom.com/share/a6834a8d802f481bad89add1e6dea42c)

### CODE REVIEW

File: api/children/childrenModel.js [hunk 1]
```diff
@@ -45,6 +45,13 @@ const updateChild = async (id, child) => {
   return await db('children').where({ id }).update(child).returning('*');
 };
 
+const updateEnrollment = async (id, enrollment) => {
+  return await db('enrollments')
```
Comment (MikeLikesCode) [inline api/children/childrenModel.js:49]: Looks solid. Should be good to go!

**Review by MikeLikesCode — APPROVED** [review]

File: api/admin/adminModel.js [hunk 2]
```diff
@@ -4,20 +4,20 @@ const getAdmins = async () => {
...
     'profiles',
     'admins.user_id',
-    'profiles.key'
+    'profiles.okta'
   );
 };
 
 const findByAdminId = async (id) => {
   return db('admins')
-    .leftJoin('profiles', 'admins.user_id', 'profiles.key')
```
Comment (bradlylewis) [inline api/admin/adminModel.js:13]: Looks like we just changed everything from 'hey' to 'okta' to match the tables right?

**Review by bradlylewis — APPROVED** [review]

File: api/admin/adminModel.js (hunk 2 above)
Comment (WillisLi) [reply api/admin/adminModel.js:13]: Yep!

**Review by CptHappyHands — APPROVED** [review]
Everything looks good to me

**Pull request merged**